README.md

```diff
@@ -171,9 +171,6 @@ Looking for conceptual tutorials and explanations? We have some additional artic
 
 - [Parameter-Efficient LLM Finetuning With Low-Rank Adaptation (LoRA)](https://lightning.ai/pages/community/tutorial/lora-llm/)
 
-## Pre-training
-
-Porting from Lit-LLaMA in progress 👷
```

Comment:
I see that we already have pretrain scripts [here](https://github.com/Lightning-AI/lit-gpt/tree/main/pretrain)